### PR TITLE
Fix/responsive sidebar

### PR DESCRIPTION
**Pull request by yuriikaterlin**
created responsive sidebar for tablet and desktop
adapted view usb-user-message with desktop and tablet sidebar 
adapted footer with sidebar

### CODE REVIEW

File: src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.html [hunk 1]
```diff
@@ -1,31 +1,33 @@
...
-            listItem.link === bellsNoneNotification && serviceUserMessages.countOfNoReadeMessages ? bellsNotification : listItem.link
-          }}"
-          alt="sidebar list item icon"
-        />
-        <a class="sidebar-list-item-link" [routerLink]="listItem.routerLink">{{ listItem.name | translate }}</a>
-        <div class="count-unread-messages" *ngIf="listItem.name === 'ubs-user.messages' && serviceUserMessages.countOfNoReadeMessages">
-          ({{ serviceUserMessages.countOfNoReadeMessages }})
-        </div>
+      <li class="sidebar-list-item" (click)="toggleSideBar()" *ngFor="let listItem of listElements">
+        <img class="sidebar-list-item-icon" src="{{ listItem.link }}" alt="sidebar list item icon" />
```
Comment (andreikazmirovich) [inline src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.html]: Change `src="{{ listItem.link }}"` to `[src]="listItem.link"`

File: src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.scss [hunk 2]
```diff
@@ -1,82 +1,89 @@
...
+    width: 15px;
+  }
 
-.count-unread-messages {
-  color: var(--ubs-primary-white);
-  margin: 0 -25px 0 -10px;
+  .count-unread-messages {
+    color: var(--ubs-primary-white);
+    margin: 0 -25px 0 -10px;
+  }
```
Comment (andreikazmirovich) [inline src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.scss:88]: You shouldn't change the indentations
Please leave the only changes related to your task

File: src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.html [hunk 3]
```diff
@@ -1,31 +1,33 @@
...
-  </mat-drawer-content>
-</mat-drawer-container>
+  </div>
+  <mat-drawer-container class="sidebar" autosize>
+    <mat-drawer #drawer class="sidebar-slider" mode="side" opened="true" disableClose="false">
+      <ul class="sidebar-list">
+        <li class="sidebar-list-item" *ngFor="let listItem of listElements" [routerLink]="listItem.routerLink" (click)="toggleSideBar()">
+          <img
+            class="sidebar-list-item-icon"
+            src="{{
```
Comment (andreikazmirovich) [inline src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.html]: [src]="..."

File: src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.html [hunk 4]
```diff
@@ -1,31 +1,33 @@
...
+              listItem.link === bellsNoneNotification && serviceUserMessages.countOfNoReadeMessages ? bellsNotification : listItem.link
+            }}"
```
Comment (andreikazmirovich) [inline src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.html]: It should be on one line

File: src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.html [hunk 5]
```diff
@@ -1,31 +1,33 @@
...
+            alt="sidebar list item icon"
+          />
```
Comment (andreikazmirovich) [inline src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.html:19]: Move this `/>` to the previous line

**Review by andreikazmirovich — CHANGES_REQUESTED** [review]

File: src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.html (hunk 1 above)
Comment (yuriikaterlin) [reply src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.html]: Done

File: src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.html (hunk 3 above)
Comment (yuriikaterlin) [reply src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.html]: Done

File: src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.html (hunk 4 above)
Comment (yuriikaterlin) [reply src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.html]: Done

File: src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.html (hunk 5 above)
Comment (yuriikaterlin) [reply src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.html:19]: Done

File: src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.scss (hunk 2 above)
Comment (yuriikaterlin) [reply src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.scss:88]: Done

**Review by andreikazmirovich — APPROVED** [review]

**Review by PereDima — APPROVED** [review]

**Pull request merged**